src/config.js

```diff
@@ -81,6 +85,11 @@ class Config {
 
         if (configuration["stablePHP"] !== undefined) {
             this.stable_version = configuration["stablePHP"];
+            this.minimum_version = this.stable_version;
+        }
+
+        if (this.versions.length > 0) {
+            this.minimum_version = this.versions[0]
```

Comment:
Are you sure they're sorted when you do this?

Reply:
since we are iterating over the list, I think yes. But I did not try that, but can do so.